.github/ISSUE_TEMPLATE/confiq.yml

```
@@ -0,0 +1,30 @@
+name: meta/repository Report
+description: for requesting changes and additions to the repo itself.
+title: "meta/ repository issue"
+labels: ["updates"]
+body:
+  - type: markdown
+    attributes:
+      value: '## Thanks for taking the time to contribute to the repository.'
```

I'd like for there to be a checkbox here which asks the user to confirm that they  have searched for similar issues.
You can ofc, have this in addition to what you wrote already if you want that :) Always nice to thank contributors!
```suggestion
        value: '## Before you continue, please search our open/closed issues to see if a similar issue has been addressed.'

  - type: checkboxes
    attributes:
      label: I have searched through the issues and didn't find my problem.
      options:
        - label: Confirm
          required: true
```